### ISSUE TITLE
digdag v0.9.41 doesn't use HTTPS on RemotePluginLoader

### ISSUE THREAD

**Issue by NomadBlacky** [treasure-data/digdag#1410]
I'm getting the following errors in my environment.

```
+transform_report_by_athena:
  _export:
    plugin:
      repositories:
        - https://jitpack.io
      dependencies:
        - pro.civitaspo:digdag-operator-athena:0.1.5
    athena:
      region: ap-northeast-1
      database: foo
      table: bar
  +prepare_database:
    athena.query>: queries/athena_create_database.sql
    preview: false
```

```
2020-05-27 03:09:37.547 +0000 [ERROR] (2045@[0:debug_wf]+report_coordinator+reports^sub+for-0=target=0=%7B%22name%22%3A^sub+load^sub+transform_report_by_athena+prepare_database) io.digdag.core.agent.OperatorManager: Task failed with unexpected error: org.eclipse.aether.resolution.DependencyResolutionException: Could not transfer artifact pro.civitaspo:digdag-operator-athena:jar:0.1.5 from/to jcenter (http://jcenter.bintray.com/): status code: 403, reason phrase: Forbidden (403)
java.lang.RuntimeException: org.eclipse.aether.resolution.DependencyResolutionException: Could not transfer artifact pro.civitaspo:digdag-operator-athena:jar:0.1.5 from/to jcenter (http://jcenter.bintray.com/): status code: 403, reason phrase: Forbidden (403)
	at com.google.common.base.Throwables.propagate(Throwables.java:241)
	at io.digdag.core.plugin.RemotePluginLoader.resolveArtifacts(RemotePluginLoader.java:167)
	at io.digdag.core.plugin.RemotePluginLoader.load(RemotePluginLoader.java:119)
	at io.digdag.core.plugin.DynamicPluginLoader.loadCache(DynamicPluginLoader.java:66)
	at io.digdag.core.plugin.DynamicPluginLoader.lambda$load$0(DynamicPluginLoader.java:54)
	at com.google.common.cache.LocalCache$LocalManualCache$1.load(LocalCache.java:4718)
	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3445)
	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2194)
	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2153)
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2043)
	at com.google.common.cache.LocalCache.get(LocalCache.java:3851)
	at com.google.common.cache.LocalCache$LocalManualCache.get(LocalCache.java:4713)
	at io.digdag.core.plugin.DynamicPluginLoader.load(DynamicPluginLoader.java:54)
	at io.digdag.core.agent.OperatorRegistry.get(OperatorRegistry.java:97)
	at io.digdag.core.agent.OperatorManager.callExecutor(OperatorManager.java:284)
	at io.digdag.core.agent.OperatorManager.runWithWorkspace(OperatorManager.java:257)
	at io.digdag.core.agent.OperatorManager.lambda$runWithHeartbeat$2(OperatorManager.java:137)
	at io.digdag.core.agent.ExtractArchiveWorkspaceManager.withExtractedArchive(ExtractArchiveWorkspaceManager.java:77)
	at io.digdag.core.agent.OperatorManager.runWithHeartbeat(OperatorManager.java:135)
	at io.digdag.core.agent.OperatorManager.run(OperatorManager.java:119)
	at io.digdag.core.agent.MultiThreadAgent.lambda$null$0(MultiThreadAgent.java:127)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.eclipse.aether.resolution.DependencyResolutionException: Could not transfer artifact pro.civitaspo:digdag-operator-athena:jar:0.1.5 from/to jcenter (http://jcenter.bintray.com/): status code: 403, reason phrase: Forbidden (403)
	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveDependencies(DefaultRepositorySystem.java:391)
	at io.digdag.core.plugin.RemotePluginLoader.resolveArtifacts(RemotePluginLoader.java:164)
	... 24 common frames omitted
Caused by: org.eclipse.aether.resolution.ArtifactResolutionException: Could not transfer artifact pro.civitaspo:digdag-operator-athena:jar:0.1.5 from/to jcenter (http://jcenter.bintray.com/): status code: 403, reason phrase: Forbidden (403)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:444)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifacts(DefaultArtifactResolver.java:246)
	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveDependencies(DefaultRepositorySystem.java:373)
	... 25 common frames omitted
Caused by: org.eclipse.aether.transfer.ArtifactTransferException: Could not transfer artifact pro.civitaspo:digdag-operator-athena:jar:0.1.5 from/to jcenter (http://jcenter.bintray.com/): status code: 403, reason phrase: Forbidden (403)
	at org.eclipse.aether.connector.basic.ArtifactTransportListener.transferFailed(ArtifactTransportListener.java:43)
	at org.eclipse.aether.connector.basic.BasicRepositoryConnector$TaskRunner.run(BasicRepositoryConnector.java:355)
	at org.eclipse.aether.util.concurrency.RunnableErrorForwarder$1.run(RunnableErrorForwarder.java:67)
	at org.eclipse.aether.connector.basic.BasicRepositoryConnector$DirectExecutor.execute(BasicRepositoryConnector.java:581)
	at org.eclipse.aether.connector.basic.BasicRepositoryConnector.get(BasicRepositoryConnector.java:249)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.performDownloads(DefaultArtifactResolver.java:520)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:421)
	... 27 common frames omitted
Caused by: org.apache.http.client.HttpResponseException: status code: 403, reason phrase: Forbidden (403)
	at org.eclipse.aether.transport.http.HttpTransporter.handleStatus(HttpTransporter.java:519)
	at org.eclipse.aether.transport.http.HttpTransporter.execute(HttpTransporter.java:322)
	at org.eclipse.aether.transport.http.HttpTransporter.implGet(HttpTransporter.java:274)
	at org.eclipse.aether.spi.connector.transport.AbstractTransporter.get(AbstractTransporter.java:59)
	at org.eclipse.aether.connector.basic.BasicRepositoryConnector$GetTaskRunner.runTask(BasicRepositoryConnector.java:447)
	at org.eclipse.aether.connector.basic.BasicRepositoryConnector$TaskRunner.run(BasicRepositoryConnector.java:350)
	... 32 common frames omitted
```

**Expected behervior**

Digdag getting the plugin from `https://jitpack.io`.

**IMO**

I don't know why can't get the plugin from jitpack, but I seem the digdag v0.9.41 doesn't use https://jcenter.bintray.com/ on RemotePluginLoader.

https://github.com/treasure-data/digdag/blob/v0.9.41/digdag-core/src/main/java/io/digdag/core/plugin/RemotePluginLoader.java#L50

**Comment by hiroyuki-sato** [comment]
Hello, @NomadBlacky 

I just copied your configuration file and execute `digdag run test.dig`
I succeed to download the plugin.

What digdag mode do you use? If you use server mode, Could you make sure the directory permission? Especially, the directory which you executed `digag server` command. Because digdag server downloads a plugin there.

```
digdag run test.dig
2020-05-28 17:44:38 +0900: Digdag v0.9.41
2020-05-28 17:44:41 +0900 [WARN] (main): Using a new session time 2020-05-28T00:00:00+00:00.
2020-05-28 17:44:41 +0900 [INFO] (main): Using session /private/tmp/c/.digdag/status/20200528T000000+0000.
2020-05-28 17:44:41 +0900 [INFO] (main): Starting a new session project id=1 workflow name=test session_time=2020-05-28T00:00:00+00:00
2020-05-28 17:44:42 +0900 [INFO] (0017@[0:default]+test+transform_report_by_athena+prepare_database): athena.query>: queries/athena_create_database.sql
```

```
find . -name '*.jar' -print
./.digdag/plugins/pro/civitaspo/digdag-operator-athena/0.1.5/digdag-operator-athena-0.1.5.jar
```

**Comment by NomadBlacky** [comment]
@hiroyuki-sato Thank you for your reply.

Sorry, I’m currently unable to reproduce this issue...
Digdag can download plugins correctly now.

> What digdag mode do you use? If you use server mode, Could you make sure the directory permission?

I use server mode.
And, I seem the directory permission is correct.

```
$ whoami
digdag

$ pwd
/home/digdag

$ ls -la
drwxr-xr-x 3 digdag digdag 4096  5月 27 23:30 .digdag

$ find . -name '*.jar' -print
./.digdag/plugins/pro/civitaspo/digdag-operator-athena/0.1.5/digdag-operator-athena-0.1.5.jar
```

**Comment by NomadBlacky** [comment]
`RemotePluginLoader` has been fixed at v0.9.42, so I'll use this one.

https://github.com/treasure-data/digdag/blob/v0.9.42/digdag-core/src/main/java/io/digdag/core/plugin/RemotePluginLoader.java#L50
